README.md

```diff
@@ -52,6 +52,47 @@ The resulting DOM is essentially:
 When a file is chosen, it will immediately be uploaded to S3.  You can listen for progress (and
 create a status bar, for example) by providing an `onProgress` function to the component.
 
+Using custom function to get signedUrl
+------------
+
+If can use custom function to get provide `signedUrl` direclty to `s3uploader` by adding `getSignedUrl` prop. The function you provide should take `file` and `callback` arguments. Callback should be called with an object containing `signedUrl` key.
+
+```javascript
+import fetch from './ApiClient';
+
+function getSignedUrl(file, callback) {
+  const client = new ApiClient();
+  const params = {
+    objectName: file.name,
+    contentType: file.type
+  };
+
+  client.get(signingUrl, { params })
```

Comment:
`signingUrl` wouldn't be defined here, so maybe change to `/my/signing/server` or something.